resources/regressions/all.json/gambit_results.json

```diff
@@ -309,156 +309,163 @@
   },
   {
     "description": "DeleteExpressionMutation",
-    "diff": "--- original\n+++ mutant\n@@ -6,7 +6,8 @@\n \n     function myIdentity(uint256 x) public pure returns (uint256) {\n \tuint256 result = 0;\n-\tfor (uint256 i = 0; i < x; i++) {\n+\t/// DeleteExpressionMutation(`i++` |==> `/* i++ */`) of: `for (uint256 i = 0; i < x; i++) {`\n+\tfor (uint256 i = 0; i < x; /* i++ */) {\n \t    result ++;\n \t}\n \treturn result;\n",
```

Comment:
what are all the changes in this file? Shouldn't it only add new mutants?

Reply:
@chandrakananandi these were changed automatically by a script i ran to update the regressions. im not sure why this happened or if it is necessary. should we check in with Ben?

Reply:
@BenTheKush is this expected?

Reply:
You can see that this makes sense: the first weird diff is for a delete expression mutation. 
* It looks like we are addingthe first new `DeleteExpressionmutation` at mutant id = 45 (which was already a delete expression mutation).
* Then the _old_ `DeleteExpressionMutant` becomes mutant id 46, which used to be `IfStatementMutation`.
* The displaced `IfStatementMutation`, which was mid=46, becomes mid=47, which was also an `IfStatementMutation`.
* The second displaced `IfStatementMutation` becomes mid=48, which was a `SwapArgumentMutation`. This now falls to mid=49, etc, etc

Reply:
we just spoke over slack. these are expected/necessary and we should keep them

Reply:
sg